AsyncDisplayKit/ASPagerNode.m

```diff
@@ -27,29 +27,34 @@ - (instancetype)init
   
   self = [super initWithCollectionViewLayout:_flowLayout];
   if (self != nil) {
-    self.view.asyncDataSource = self;
-    self.view.asyncDelegate = self;
-    
-    self.view.pagingEnabled = YES;
-    self.view.allowsSelection = NO;
-    self.view.showsVerticalScrollIndicator = NO;
-    self.view.showsHorizontalScrollIndicator = NO;
-    
-    ASRangeTuningParameters tuningParams = { .leadingBufferScreenfuls = 1.0, .trailingBufferScreenfuls = 1.0 };
-    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypePreload];
-    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypeRender];
   }
   return self;
 }
 
-- (void)reloadData
+- (void)didLoad
 {
-  [self.view reloadData];
+  [super didLoad];
+  
+  self.view.asyncDataSource = self;
+  self.view.asyncDelegate = self;
+  
+  self.view.pagingEnabled = YES;
+  self.view.allowsSelection = NO;
+  self.view.showsVerticalScrollIndicator = NO;
+  self.view.showsHorizontalScrollIndicator = NO;
+  
+  ASRangeTuningParameters preloadParams = { .leadingBufferScreenfuls = 2.0, .trailingBufferScreenfuls = 2.0 };
+  ASRangeTuningParameters renderParams = { .leadingBufferScreenfuls = 1.0, .trailingBufferScreenfuls = 1.0 };
+  [self setTuningParameters:preloadParams forRangeType:ASLayoutRangeTypePreload];
+  [self setTuningParameters:renderParams forRangeType:ASLayoutRangeTypeRender];
 }
 
-- (void)setTuningParameters:(ASRangeTuningParameters)tuningParameters forRangeType:(ASLayoutRangeType)rangeType
+#pragma mark - Helpers
+
+- (void)scrollToPageAtIndex:(NSInteger)index animated:(BOOL)animated
 {
-  [self.view setTuningParameters:tuningParameters forRangeType:rangeType];
+  NSIndexPath *indexPath = [NSIndexPath indexPathForItem:index inSection:0];
+  [self.view scrollToItemAtIndexPath:indexPath atScrollPosition:UICollectionViewScrollPositionLeft animated:animated];
```

Comment:
Hmm, interesting - I see the logic behind left, but I think most clients will use this with some spacing and intending to be centered onscreen...so maybe we should initially default to center and see how it goes.